packages/pages/sites-config/features.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Is this file supposed to be here, or generated when Pages is run?

Reply:
Nope. Not sure how that happened. Deleted.